change/@ni-nimble-components-df1cc26c-9976-477a-8fa0-f9b72492eaa0.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Add keyboard navigation functionality to the table component",
```

Comment:
I think this is a bug, but looks like after entering the table, tabbing out and interacting with a different control, then re-tabbing into the table the focus target is different. I think both time entering the table I was expecting the focus target to be the same (either always first row or always first interactable header part)


![reenter table2](https://github.com/ni/nimble/assets/1588923/17caa6f2-b2f6-4875-aa4f-33785f71d545)

Reply:
Talked with Malcolm, this was my misunderstanding, it's working as expect jumping back to the last focused row